common/settings.h

```diff
@@ -126,10 +126,13 @@ struct Settings {
 
   std::string temp_directory_path;
   std::vector<std::string> dart_flags;
+
+#if OS_FUCHSIA
```

Comment:
The Fuchsia runner is in the engine tree. Can that be patched right away as well?

Reply:
Yeah, I think we can just patch the fuchsia code.  Left a comment explaining how below.

Reply:
Done